src/controller/expenses.controller.js

```diff
@@ -0,0 +1,122 @@
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const { userId, from, to, categories } = req.query;
+  const expenses = await expensesService.getAllExpenses();
```

Comment:
This line retrieves all expenses without considering the user's query. This could be a performance issue if the database contains a large number of expense records. It would be more efficient to apply the filters in the database query, rather than retrieving all expenses and then filtering them in the application code.